crates/compiler/load/tests/test_reporting.rs

```diff
@@ -10856,6 +10855,71 @@ In roc, functions are always written as a lambda, like{}
     //     "#
     // );
 
+    test_report!(
+        empty_new_record_builder,
+        indoc!(
+            r#"
+            { a <- }
+            "#
+        ),
+        @r#"
+    ── EMPTY RECORD BUILDER in /code/proj/Main.roc ─────────────────────────────────
+    
+    This record builder has no fields:
+    
+    4│      { a <- }
+            ^^^^^^^^
+    
+    Note: We need at least two fields to combine their values into a record.
+    "#
+    );
+
+    test_report!(
+        single_field_new_record_builder,
+        indoc!(
+            r#"
+            { a <-
+                b: 123
+            }
+            "#
+        ),
+        @r#"
+    ── NOT ENOUGH FIELDS IN RECORD BUILDER in /code/proj/Main.roc ──────────────────
+
+    This record builder only has one field:
+    
+    4│>      { a <-
+    5│>          b: 123
+    6│>      }
+    
+    Note: We need at least two fields to combine their values into a record.
+    "#
+    );
+
+    test_report!(
+        optional_field_in_new_record_builder,
+        indoc!(
+            r#"
+            { a <-
+                b: 123,
+                c? 456
+            }
+            "#
+        ),
+        @r#"
+    ── OPTIONAL FIELD IN RECORD BUILDER in /code/proj/Main.roc ─────────────────────
+    
+    Optional fields are not allowed to be used in record builders.
+    
+    4│       { a <-
+    5│           b: 123,
+    6│>          c? 456
+    7│       }
+    
+    Note: Record builders can only have required values for their fields.
+    "#
+    );
+
```

Comment:
I think we reserve "Note:" for additional clarification, but here is being used for the main issue. I'd drop it.

Reply:
Also, in most of our errors, we use singular first person pronouns.